lib/recog/db_manager.rb

```diff
@@ -13,8 +13,12 @@ def initialize(path = DefaultDatabasePath)
   end
 
   def load_databases
-    Dir[self.path + "/*.xml"].each do |dbxml|
-      self.databases << DB.new(dbxml)
+    if File.directory?(self.path)
+      Dir[self.path + "/*.xml"].each do |dbxml|
```

Comment:
Hmm.  Thinking about this while looking at the output of your utility that is exercising this code and I think you might need to ensure that the files are always returned in the same order.  From what I've read, `Dir.glob` will not guarantee order.  So, in the event of two XML files with the same preference, there is no guarantee as to the order they will be loaded.  This will allow odd ordering problems that might exist between different operating systems to cause the shipped fingerprints to be evaluated differently.